### PR TITLE
Button

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
- Add buttons elements 
- Add button stories
- Add styles
- Add globals CSS 
- Style buttons

### CODE REVIEW

File: next-env.d.ts [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+
+/* declare module "*.css"; */
```
Comment (phlgr) [inline next-env.d.ts:4]: Why have you added this?

Comment (SilviaBarcellini) [reply next-env.d.ts:4]: I was trying to fix the  import bug.  It didn't work, so I'd rather delete this line actually.

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀

**Review by sweisz — COMMENTED** [review]
GJ 👍

File: styles/globals.css [hunk 2]
```diff
@@ -1,16 +1,31 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@900&family=Paytone+One&family=Quicksand&display=swap");
```
Comment (mhstrkmp) [inline styles/globals.css:1]: Are there some fonts missing in the import? (Ubuntu, Fira Sans, Droid Sans)

**Review by mhstrkmp — COMMENTED** [review]
lgtm 👍

File: styles/globals.css (hunk 2 above)
Comment (SilviaBarcellini) [reply styles/globals.css:1]: I imported these fonts from Google Fonts. Other fonts were already there by default (lines 7,8)

**Pull request merged**